setup.py

```diff
@@ -37,8 +37,8 @@
                         'six>=1.9.0',
                         'pyyaml',
                         'h5py',
-                        'keras_applications>=1.0.6',
-                        'keras_preprocessing>=1.0.5'],
+                        'keras_applications>=1.0.7',
+                        'keras_preprocessing>=1.0.6'],
```

Comment:
1.0.7 for KP please
We had a pandas import that was not handled correctly.

Reply:
I'll make a test for this kind of thing. We got lucky this PR was not merged.

Reply:
Still problematic since it can have affected users who did a fresh install of Keras in the past 12 hours (since the current setup.py lists `keras_preprocessing >=1.0.5` which would then have been 1.0.6 by default).

I re-released on PyPI. Note that when we make a tag on git we should make sure to first update the version number in the code (keep the code in sync with tags).

Reply:
Yes it is problematic. We have informed the users affected in the issue. 

Should we use hardcoded versions instead of >= ?

Reply:
Unfortunately we cannot use exact versions, because these packages are used as dependencies in both Keras and TF, and each is likely to require a different version, which would make concurrent installations impossible (which would be a total wreck).

Reply:
For the sake of stability we'll start using exact versions for TF, and `>=` for Keras. The expectation being that Keras will be upgraded earlier, and will be easier to update in case of issue.

Reply:
That's possibly problematic as well.

In any case, we should strengthen our unit testing and have integration tests to catch precisely this type of issue (dependencies, compatibility with the latest Keras on PyPI, compatibility with the latest TF).